docs/jwst/extract_1d/description.rst

```diff
@@ -450,6 +450,38 @@ where `flux` is the extracted spectral data, and the data are from channel 4 for
 
 Extraction for NIRISS SOSS Data
 -------------------------------
-For NIRISS SOSS data, the two spectral orders overlap slightly, so a specialized extraction
-algorithm known as ATOCA (Algorithm to Treat Order ContAmination) is used...
-Link paper 
+For NIRISS SOSS data, spectral orders 1 and 2 overlap slightly at longer wavelengths so a specialized extraction
+algorithm known as ATOCA (Algorithm to Treat Order ContAmination, Darveau-Bernier et al., 2022) is made available in
+the ``extract_1d`` step. This routine constructs a linear model of each pixel on the detector and treats the underlying
+incident spectrum as a free variable to simultaneously extract the cross-contaminated spectra, which are accurate to
+10ppm over the full spectral range.
+
+The algorithm uses a wavelength solution, a spectral throughput, a spectral resolution, and a spatial throughput for
+both orders to determine the flux contribution from each order falling on a given pixel. Most of these references are
+determined by analysis of on-sky data and supplied to the algorithm via the `pastasoss` and `spec_profile` reference
+files. The exception is the `spec_kernel` reference file which supplies the convolution kernels used in the extraction,
+determined from monochromatic PSFs generated by WebbPSF. The `pastasoss` reference file predicts the trace centroids,
+taking into account the small rotations of the trace introduced by the slight visit-to-visit offsets of the GR700XD
+grism in the optical path.
+
+Having constructed a model of the intensity at each pixel using the reference file inputs, a chi-square minimization is
+used to fit the pixel model to the observations, weighted by the uncertainty. However, since a solution to this system
+of equations is highly degenerate, a Tikhonov regularization (Tikhonov 1963) is performed. The goal here is to find the
+smoothest solution for the flux that fits the observations within the measured uncertainties.
+
+The resulting spectral trace solutions are actually at a higher resolution than the observed data since an oversampled
+wavelength grid is used by the ATOCA algorithm for decontamination. These results are then reconvolved onto the native
+wavelength grid before the 1D spectra for each order are extracted.
+
+There are a number of keyword arguments accessible to the user to customize the ``extract_1d`` for SOSS data.
```

Comment:
I think this section could be removed because it should be redundant with https://jwst-pipeline.readthedocs.io/en/latest/jwst/extract_1d/arguments.html 

I'd suggest having a look there and making whatever updates to the soss arguments you see fit

Reply:
Sure thing. Yeah, it's definitely redundant so I'll just take it out.